### PR TITLE
Generalize HiveExecutor constructor method to accept custom executor …

### PR DESCRIPTION
**Pull request by chetcode**
1. Added a new HiveTaskExecutor constructor to accept custom executor clients
2. Make `NewQuboleStatus` function public

### CODE REVIEW

**Comment by chetcode** [comment]
👀 @EngHabu

File: go/tasks/v1/qubole/hive_executor.go [hunk 1]
```diff
@@ -562,6 +563,16 @@ func NewHiveTaskExecutorWithCache(ctx context.Context) (*HiveExecutor, error) {
 	return &hiveExecutor, nil
 }
 
+func NewHiveTaskExecutor(ctx context.Context, executorId string, executorClient client.QuboleClient) (*HiveExecutor, error) {
```
Comment (chetcode) [inline go/tasks/v1/qubole/hive_executor.go]: make the old call the new one

Comment (chetcode) [reply go/tasks/v1/qubole/hive_executor.go]: done!

File: go/tasks/v1/qubole/qubole_work.go [hunk 2]
```diff
@@ -43,6 +43,8 @@ type QuboleWorkItem struct {
 	Query string `json:"query,omitempty"`
 
 	TimeoutSec uint32 `json:"timeout,omitempty"`
+
+	JobUri string `json:"job_uri,omitempty"`
```
Comment (EngHabu) [inline go/tasks/v1/qubole/qubole_work.go]:
```suggestion
	CommandUri string `json:"command_uri,omitempty"`
```

**Pull request merged**